ansible/roles/ocp4-workload-sso-cloud-native-apps/tasks/post_workload.yml

```diff
@@ -0,0 +1,48 @@
+---
+- name: Verify everything is deployed correctly
+  include_tasks: verify-workload.yaml
+
+# Implement your Post Workload deployment tasks here
+- name: output workshop info
+  debug:
```

Comment:
Should be changed to use the `agnosisticd_user_info` module. Otherwise these message won't show up in a Babylon deployment.

Example: https://github.com/redhat-cop/agnosticd/blob/development/ansible/roles_ocp_workloads/ocp4_workload_shared_bastion_access/tasks/post_workload.yml#L27-L56